doc/reference/reference_lua/net_box.rst

```diff
@@ -221,6 +247,7 @@ Below is a list of all ``net.box`` functions.
         conn = net_box.connect('localhost:3301')
         conn = net_box.connect('127.0.0.1:3302', {wait_connected = false})
         conn = net_box.connect('127.0.0.1:3303', {reconnect_after = 5, call_16 = true})
+        conn = net_box.connect('127.0.0.1:3304', {required_protocol_version = 4, required_protocol_features = {'transactions'}, })
```

Comment:
For the ``required_protocol_features``, it's better to have an example with two or more values. The feature description says: You can specify one or more ``net.box`` features. 
But the syntax how to specify them is not clear. The example should make it clear.

Reply:
Fixed.